### ISSUE TITLE
Stereo parameter?

### ISSUE THREAD

**Issue by rebotnix** [descriptinc/descript-audio-codec#25]
Hi, great project, i encoded a 2 CH input audio file with the 44khz model, all works fine, but it seems only to use 1 channel. 

Decoded wave shows:
  Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 44100 Hz, 1 channels, s16, 705 kb/s

How can i set that encoding and encoding uses 2 channels for stereo encoding and decoding?

thx

**Comment by pseeth** [comment]
Hello there! I have a PR in the works that supports this, along with chunked inference for compressing without running out of GPU memory. Will have it merged soon!

**Comment by rebotnix** [comment]
Really cool @pseeth Will try to test as soon as it is merged and response with the results. Thanks.

**Comment by pseeth** [comment]
This is now merged in the most recent DAC 1.0.0. Let me know if you have any issues! Use the `model.compress` and `model.decompress` functions.

**Comment by listener17** [comment]
@pseeth is my understanding correct that if I take the 44.1 kHz model, and encode-decode a stereo audio file, it's encoded at 8 kbps?

**Comment by drscotthawley** [comment]
> Hello there! I have a PR in the works that supports this, along with chunked inference for compressing without running out of GPU memory. Will have it merged soon!
> This is now merged in the most recent DAC 1.0.0. Let me know if you have any issues! Use the model.compress and model.decompress functions.

@pseeth Did these truly get merged -- or if so, how do we enable these capabilities?  Because I'm getting an error when trying stereo, AND I'm running out of GPU RAM. 

Trying `model.compress(x)` produces an error: 
```python
AttributeError: 'Tensor' object has no attribute 'sample_rate'
```

Looking forward to trying this!
